docs/development/contribution-documentation/documentation-style-guide/README.md

```diff
@@ -410,7 +409,25 @@ When you take screenshots:
 
 - Be consistent: Coordinate screenshots with the other screenshots already on a documentation page. For example, if other screenshots include the left sidebar, include the sidebar in all screenshots.
 
-  
+
+### Highlight specific areas and add numbered labels
+
+You can highlight a specific area of a screenshot if you would like to draw the reader's attention to it. Additionally, you can add numbered labels so you can refer to specific parts of the screenshot in the documentation text. These should, however, be used sparingly, since any future changes to the interface require the highlighted areas and labels to be manually be added to each updated screenshot.
```

Comment:
```suggestion
You can highlight a specific area of a screenshot to draw the reader's attention to it. Additionally, add numbered labels to refer to specific parts of the screenshot in the documentation text. These should, however, be used sparingly, since any future changes to the interface require the highlighted areas and labels to be manually added to each updated screenshot.
```